xdist/workermanage.py

```diff
@@ -201,19 +201,19 @@ def make_reltoroot(roots, args):
     return result
 
 
-class SlaveController(object):
+class workerController(object):
     ENDMARK = -1
 
     def __init__(self, nodemanager, gateway, config, putevent):
         self.nodemanager = nodemanager
         self.putevent = putevent
         self.gateway = gateway
         self.config = config
-        self.slaveinput = {'slaveid': gateway.id,
-                           'slavecount': len(nodemanager.specs)}
+        self.workerinput = {'workerid': gateway.id,
```

Comment:
Please add an alias here for backward compatibility:

```python
# deprecated name, backward compatibility only
self.slaveinput = workerinput  
```

Reply:
Fixed

Reply:
this dict also need the old names (`slaveid` and `slavecount`)

and we likely need to keep those for around 5 years at least

i propose adding as is and later on following up with a dict subclass that gives deprecation warnings in a few years

Reply:
Ah good catch, I missed those. 👍

Reply:
@nicoddemus @RonnyPfannschmidt fixed